### PR TITLE
chore: Change fvm config.json to .fvmrc

### PR DESCRIPTION
**Pull request by riscait**
## 🙌 What's Done

- Change fvm config.json to .fvmrc
- Remove Japanese README

## ✍️ What's Not Done

-

### CODE REVIEW

File: .github/workflows/flutter-app-code-check.yml
Comment (riscait) [inline .github/workflows/flutter-app-code-check.yml:1]: Omission of change.

File: Makefile
Comment (riscait) [inline Makefile:1]: Changed to installation using Homebrew.

File: README-ja.md
Comment (riscait) [inline README-ja.md:1]: Deleted due to lack of maintenance.

File: README.md [hunk 1]
```diff
@@ -15,40 +12,6 @@ README in other languages
...
-This package is responsible for functions (responsibilities) related to local notification display and schedule registration.
-
-Currently using `flutter_local_notifications`.
-
-### `remote_parameter_fetcher`
-
-This package is responsible for the function (responsibility) of retrieving remotely configured parameters and providing them in a specified type.
-
-Currently using Firebase Remote Config.
-
```
Comment (riscait) [inline README.md:51]: Removed description of internal packages to be removed.

File: README.md [hunk 2]
```diff
@@ -68,36 +31,6 @@ Change the app ID or app name to start building it as a separate app right away.
...
-## Router
-- [go_router](https://pub.dev/packages/go_router)
-
-## `flutter_app` Project settings
-|Category|Description|Codes|
-|---|---|---|
-| [FVM](https://github.com/leoafarias/fvm) | Flutter Version Management | [.fvm/fvm_config.json](/.fvm/fvm_config.json) |
-| Dart | Dart version | [pubspec.yaml](/packages/flutter_app/pubspec.yaml) |
-| Dart | Lint / Analyze | [analysis_options.yaml](/packages/flutter_app/analysis_options.yaml) |
```
Comment (riscait) [inline README.md:91]: Go to README on the application side.

**Review by k-nkmr — APPROVED** [review]
LGTM

**Review by naipaka — APPROVED** [review]
LGTM

**Pull request merged**